lib/driver.js

```diff
@@ -771,12 +769,63 @@ class XCUITestDriver extends BaseDriver {
     }
     log.debug(`Setting initial orientation to '${orientation}'`);
     try {
-      await this.proxyCommand('/orientation', 'POST', {orientation});
+      await this._executeCommandWithTimeout(B.promisify(this.proxyCommand), 'setScreenOrientation', '/orientation', 'POST', {orientation});
```

Comment:
Hmmm. That is a difficult one. I think `appium-base-driver` would need to be modified to export the mapping, which would need to be generated from the methods map. https://github.com/appium/appium-base-driver/blob/master/lib/mjsonwp/routes.js#L7